cot/tests/project.rs

```diff
@@ -11,7 +12,7 @@ use cot::{App, AppBuilder, Body, Project, StatusCode, reverse};
 #[cfg_attr(miri, ignore)] // unsupported operation: can't call foreign function `sqlite3_open_v2`
 async fn cot_project_router_sub_path() {
     async fn hello(_request: Request) -> cot::Result<Response> {
-        Ok(Response::new_html(StatusCode::OK, Body::fixed("OK")))
+        Ok(Html::new("OK").into_response())
```

Comment:
Can we remove `into_response()` here and just change the return type?

Reply:
Not everywhere - in some cases where there are two differently typed responses returned, it has to be `Response`, but in this case and a few others, I should be able to change it to `Html`.